lib/std/time.zig

```diff
@@ -219,3 +224,1121 @@ test "Timer" {
     timer.reset();
     testing.expect(timer.read() < time_1);
 }
+
+
+pub const Weekday = enum(u4){
+    Monday = 1,
```

Comment:
Why start at `1`?  (I don't mind it; but would like your rationale)

As a counterpoint, `tm_wday` in `struct tm` is numbered 0-6.

Reply:
ISO 8601 specifies weekday number to be 1-7 starting with Monday.